test/factory-test.js

```diff
@@ -605,4 +610,27 @@ describe('factory', function() {
     });
   });
 
+  describe('extend option', function() {
+    it('inherits properties of parent, and also has its own', function(done){
+      factory.build('interestingPerson', function(err, somePerson){
+        (somePerson instanceof Person).should.be.true;
+        somePerson.age.should.eql(25);
+        expect(somePerson.interests).to.contain('running', 'chess');
```

Comment:
I didn't know it had an 'include' or 'contain' type of checker. Next commit will contain your recommendation.